### DISCUSSION TITLE
Styled Components does not work with DayPickerInput

### DISCUSSION THREAD

**Discussion by jgatjens** [gpbl/react-day-picker#1121] [Support] [answered]
https://codesandbox.io/s/r4x57p6x2o

**Comment by trietlam** [comment]
I can do it with a wrapper, not sure if it's proper way.
https://codesandbox.io/s/7zoy5293v0

**Comment by jgatjens** [comment]
I did pretty much the same way but with scss. how I did not think this way.
👍

**Comment by kirillku** [comment]
To avoid having wrapper, you can pass the right `className` to `classNames` prop of `DayPickerInput`. Check [styling-normal-react-components section](https://www.styled-components.com/docs/advanced#styling-normal-react-components) in `styled-components` docs.

Updated codesandbox: https://codesandbox.io/s/p3427n1m8j

**Comment by matiasmenker** [comment]
Update fork because the wrapper does not work with **refs**, so i add forwardRef:

https://codesandbox.io/s/9lkmj6mlvp?fontsize=14

**Comment by brunodasilvalenga** [comment, marked as answer]
Hey, you can create a wrapper and replace the existing css, like this:


```

const WrapperStyled = styled.div`
  &&& {
    .DayPicker,
    .DayPicker-wrapper {
      :focus {
        outline: none;
      }
    }

    .DayPicker-Weekday {
      color: #6adc91;
      font-size: 18px;
      font-family: 'Montserrat-SemiBold';
    }
}
`
```

```
const Calendar = () => {
return (
<WrapperStyled>
      <DayPicker
        {...props}
        numberOfMonths={2}
        selectedDays={selectedDays}
        navbarElement={<Navbar />}
        captionElement={<Caption />}
        onDayClick={handleDayClick}
      />
    </WrapperStyled>
)
}```
